courses/management/commands/bulk_load_topics.py

```diff
@@ -0,0 +1,98 @@
+"""Management command to bulk load topics from a CSV into the system
+
+Arguments:
+* --file <filename> - Path to the CSV file containing the topics
+
+--file must be a valid CSV file. The command assumes that the file would have a header row containing all the topics/subtopics.
+The sample file format can be seen below:
+
+topic       | topic         | topic
+subtopic    | subtopic      | subtopic
+subtopic    | subtopic      | subtopic
+
+"""
+
+import csv
+from pathlib import Path
+
+from django.core.management import BaseCommand, CommandError
+
+from courses.models import CourseTopic
+
+
+def create_topic(name=None, parent=None):
+    """Helper method to create topics based on topic/subtopic relationship"""
+    name = name.strip()
+    if not parent:
+        return CourseTopic.objects.get_or_create(name=name)
+    parent = parent.strip()
+    parent_topic = CourseTopic.objects.get(name=parent, parent__isnull=True)
+    return CourseTopic.objects.get_or_create(name=name, parent=parent_topic)
+
+
+class Command(BaseCommand):
+    """The command reads a list of topics/subtopics from a CSV file and loads them into system"""
+
+    help = "Bulk loads the topics from a CSV file"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--file",
+            type=str,
+            help="Path to the CSV file containing topics",
+            required=True,
+            dest="topics_file",
+        )
+
+    def handle(self, *args, **options):  # noqa: ARG002
+        """Handle command execution"""
+
+        file_path = options["topics_file"].strip()
+
+        if not file_path or not file_path.endswith(".csv"):
+            raise CommandError("The command can handle only CSV files")  # noqa: EM101
+
+        if not file_path or not Path(file_path).exists():
```

Comment:
No need for file_path check as already checked above.
```suggestion
        if not Path(file_path).exists():
```